addons/event/models/event_event.py

```diff
@@ -494,6 +497,16 @@ def _compute_ticket_instructions(self):
                is_html_empty(event.event_type_id.ticket_instructions):
                 event.ticket_instructions = event.event_type_id.ticket_instructions
 
+    @api.depends('address_id')
+    def _compute_venue_inline(self):
+        """Use venue address if available, otherwise its name, finally ''. """
+        for event in self:
+            if (event.address_id.contact_address or '').strip():
+                event.venue_inline = ', '.join(
```

Comment:
Can't it be used in ``_get_event_resource_urls`` ? It uses spaces instead of commas but hey.